### DISCUSSION TITLE
Two Android App issues that may belong here

### DISCUSSION THREAD

**Discussion by s38b35M5** [bookwyrm-social/bookwyrm#3250] [Q&A] [answered]
Can I get a ruling on if these two issues I created below (from the bookwyrm android app github) are actually Bookwyrm-proper and not related to the android app?

I don't want to waste resources creating issues here if they aren't in fact problem with the main project and not the app..

[FEATURE - Activity/State Visibility - Change Batch](https://github.com/PrivacyDragon/Bookwyrm_Android/issues/7#issuecomment-1906231475)

[Bug - Timestamp of book activities incorrect or changed](https://github.com/PrivacyDragon/Bookwyrm_Android/issues/8#issuecomment-1906236132)

**Comment by mouse-reeve** [comment, marked as answer]
I believe those are both bugs on this repos end, not app bugs
